packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -0,0 +1,164 @@
+
+## Authentication flow using ACR values
+
+<div class="three-quarter">
+
+![Flow diagram that displays the back and forth between the client, resource server, and the authorization server](/img/auth/step-up-authentication-using-acr-values-flow.png)
+
+</div>
+
+At a high-level, this flow has the following steps:
+
+1. Per your use case, include the `acr_values` predefined parameter value in the authentication request.
+
+2. The authentication scenarios required by the [grant type](/docs/guides/implement-grant-type/authcode/main/) authenticate the user in accordance with the predefined `acr_values` parameter value used in the authorization request.
+
+3. When the authentication flow completes, the authorization server returns an access token and/or an ID token to the client that contains the `acr` claim.
+
+4. The client requests access to the protected resource and presents the new access token.
+
+5. The resource server evaluates the assurance level of the access token against its requirements and then returns the protected resource.
+
+## Add the parameter value to the auth request
+
+The following is an example authorization request using the `urn:okta:loa:1fa:any` predefined `acr_values` parameter value. In this example, the Authorization Code [grant type](/docs/guides/implement-grant-type/authcode/main/) is used.
+
+**Request**
+
+```bash
+https://${yourOktadomain}/oauth2/default/v1/authorize?client_id={clientId}
+&response_type=code
+&scope=openid
+&redirect_uri=http://${yourOktadomain}/authorization-code/callback
+&state=296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
+&urn:okta:loa:1fa:any
+```
+
+The authorization code is returned in the response. And then the request is made to the `/token` endpoint to exchange the authorization code for an ID token and an access token. See the [Authorization Code grant type](/docs/guides/implement-grant-type/authcode/main/#grant-type-flow) for a diagram of this flow.
+
+**Response**
+
+The tokens are truncated for brevity.
+
+```json
+{
+   "token_type": "Bearer",
+   "expires_in": 3600,
+   "access_token": "eyJraW…..vSqrH1Q",
+   "scope": "openid",
+   "id_token": "eyJraW…..WlB8Y9pQ"
+}
+```
+
+To check the returned ID token payload, copy the values and paste them into any JWT decoder (for example: `<https://jwt.io/>`). Using a JWT decoder, confirm that the token contains the `acr` claim.
+
+**ID token**
+
+```json
+{
+  "sub": "00u47ijy7sRLaeSdC0g7",
+  "ver": 1,
+  "iss": "https://{yourOktadomain}/oauth2/default",
+  "aud": "0oa48e74ox4t7mQJX0g7",
+  "iat": 1660084990,
+  "exp": 1660088590,
+  "jti": "ID.OrcvMtuNx4c05Rw_QxFE_fLPBYu6HCVjDJD3nslySms",
+  "amr": [
+    "pwd"
+  ],
+  "idp": "00o47ijbqfgnq5gj00g7",
+  "auth_time": 1660084953,
+  "at_hash": "YxTPvQn3gQGwjD4yZGqXRg"
+}
+```
+
+**Access token**
```

Comment:
token missing acr_value

Reply:
yes, I still need to update all of the examples once the change is in Trex.